cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -43,10 +43,26 @@ extern "C" {
  * @brief   Available peripheral buses
  */
 enum {
```

Comment:
The F0 has an APB2 bus...

I head these in at the beginning, but decided against the other busses (AHB, etc) for a simple reason: we have only for the APB[1|2] busses the 'duality', that instances of peripherals are connected to any of them. And only in these cases we need some programmatical mean to set a bit in one of two possible registers. Same goes for selecting bus frequencies.

For all peripherals, that are always connected to the same bus (e.g. GPIO ports), we can set the effected registers directly, so there is no need to include them in these functions here.